dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilderUtility.h

```diff
@@ -16,6 +20,24 @@ namespace AppNotificationBuilder
     using namespace winrt::Microsoft::Windows::AppNotifications::Builder;
 }
 
+inline std::unordered_map<wchar_t, std::wstring> GetXmlEscapeEncodings()
```

Comment:
Why return the whole array? Instead, let this API take in a char as an argument and return a string.

Call it 
std::wstring GetXmlEscapeEncoding(const wchar_t& charToEncode)